tests/EntityFrameworkCore.Projections.Generator.Tests/ProjectionExpressionGeneratorTests.cs

```diff
@@ -380,48 +323,20 @@ public int Foo()
 ");
 
             var result = RunGenerator(compilation);
-
-            Assert.Single(result.Diagnostics);
+            return Verifier.Verify(result);
         }
 
         #region Helpers
 
-        Compilation CreateCompilation(string source, bool expectedToCompile = true)
+        Compilation CreateCompilation(string source)
         {
             var references = Basic.Reference.Assemblies.NetStandard20.All.ToList();
             references.Add(MetadataReference.CreateFromFile(typeof(ProjectableAttribute).Assembly.Location));
 
-            var assemblyPath = Path.GetDirectoryName(typeof(object).Assembly.Location);
-
-            var compilation = CSharpCompilation.Create("compilation",
+            return CSharpCompilation.Create("compilation",
                 new[] { CSharpSyntaxTree.ParseText(source) },
                 references,
                 new CSharpCompilationOptions(OutputKind.DynamicallyLinkedLibrary));
-
-#if DEBUG
```

Comment:
The following removed diagnostics reporting logic was actually useful while authoring a new test as it would tell inform me when there is a typo in my source string. Any objection against reverting this change?

Reply:
reverted :)